data/strings/types_strings.txt

```diff
@@ -9193,6 +9193,318 @@
     zh-Hans = 物理治疗师
     zh-Hant = 物理治療師
 
+  [type.healthcare.alternative]
+    en = Alternative medicine
+    ar = الطب البديل
+    be = Альтэрнатыўная медыцына
+    bg = Алтернативна медицина
+    cs = Alternativní medicína
+    da = Alternativ medicin
+    de = Alternative Medizin
+    el = Εναλλακτική ιατρική
+    es = Medicina alternativa
+    et = Alternatiivne meditsiin
+    eu = Medikuntza alternatiboa
+    fa = طب جایگزین
+    fi = Vaihtoehtoinen lääketiede
+    fr = Médecine alternative
+    he = רפואה אלטרנטיבית
+    hu = Alternatív gyógyászat
+    id = Pengobatan alternatif
+    it = Medicina alternativa
+    ja = 代替医療
+    ko = 대체 약품
+    mr = पर्यायी औषध
+    nb = Alternativ medisin
+    nl = Alternatieve geneeskunde
+    pl = Medycyna alternatywna
+    pt = Medicina alternativa
+    pt-BR = Medicina alternativa
+    ro = Medicină alternativă
+    ru = Альтернативная медицина
+    sk = Alternatívna medicína
+    sv = Alternativ medicin
+    sw = Dawa mbadala
+    th = การแพทย์ทางเลือก
+    tr = Alternatif tıp
+    uk = Альтернативна медицина
+    vi = Liều thuốc thay thế
+    zh-Hans = 替代药物
+    zh-Hant = 替代藥物
+
+  [type.healthcare.audiologist]
+    en = Audiologist
+    ar = السمعيات
+    be = Аўдыёлаг
+    bg = Аудиолог
+    cs = Audiolog
+    da = Audiolog
+    de = Audiologe
+    el = Ακουολόγος
+    es = Audiólogo
+    et = Audioloog
+    eu = Audiologoa
+    fa = شنوایی شناس
+    fi = Audiologi
+    fr = Audiologiste
+    he = אודיולוג
+    hu = Audiológus
+    id = Audiologis
+    it = Audiologo
+    ja = オーディオロジスト
+    ko = 청력학자
+    mr = ऑडिओलॉजिस्ट
+    nb = Audiolog
+    nl = Audioloog
+    pl = Audiolog
+    pt = Audiologista
+    pt-BR = Fonoaudiólogo
+    ro = Audiolog
+    ru = Аудиолог
+    sk = Audiológ
+    sv = Audiolog
+    sw = Mtaalamu wa kusikia
+    th = นักโสตสัมผัสวิทยา
+    tr = Odyolog
+    uk = Аудіолог
+    vi = chuyên gia thính học
+    zh-Hans = 听力学家
+    zh-Hant = 聽力學家
+
+  [type.healthcare.blood_donation]
+    en = Blood donation
+    ar = تبرع بالدم
+    be = Донарства крыві
+    bg = Даряване на кръв
+    cs = Darování krve
+    da = Bloddonation
+    de = Blutspende
+    el = Αιμοδοσία
+    es = Donación de sangre
+    et = Veredoonorlus
+    eu = Odol ematea
+    fa = اهدای خون
+    fi = Verenluovutus
+    fr = Don du sang
+    he = תרומת דם
+    hu = Véradás
+    id = Donor darah
+    it = Donazione di sangue
+    ja = 献血
+    ko = 헌혈
+    mr = रक्तदान
+    nb = Bloddonasjon
+    nl = Bloeddonatie
+    pl = Oddawanie krwi
```

Comment:
`pl = Krwiodawstwo`, though I feel like "Blood donation" is a little ambiguous, wouldn't something like "Blood donation center"/"Blood donation facility" be a better name for the tag?

Reply:
The iD editor calls it a `Blood Donor Center` (though I personally prefer `Blood Donation Center`, because the iD name makes it seem like it's a place for gatherings of blood donors instead of for donating blood, especially when translated)

Reply:
Good point and I totally agree.
I will change it to "Blood Donation Center" :+1:

Reply:
Renaming done in https://github.com/organicmaps/organicmaps/pull/4053/commits/dad75fe91b511fd26108fab402f7fc0a38781fe2